test/integration/proportion-Encounter-tuple/proportion-Encounter-tuple.test.ts

```diff
@@ -0,0 +1,43 @@
+import { calculate } from '../../../src/calculation/Calculator';
```

Comment:
I'm a bit confused as to why we're testing the pretty results via integration testing. Is there something specific here that isn't replicatable via hardcoded statement results or something that would work within a unit test instead? This feels more like a use case for a unit test as opposed to creating a whole measure bundle just to test pretty results